mxnet/inference/docker/1.7.0/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,129 @@
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="3"
```

Comment:
This should be major version 1, as this is the first release of 1.7.0 containers

Reply:
good catch. updated